pkg/machines/components/networks/networkOverviewTab.jsx

```diff
@@ -0,0 +1,129 @@
+    const id = `${idPrefix}-${family}-dhcp-host-${index}`;
+
+    let hostVals = [];
+    if (host.name)
+        hostVals.push(_("Name: ") + host.name);
+    if (host.mac) // MAC for ipv4, ID for ipv6
+        hostVals.push("MAC: " + host.mac);
+    else if (host.id)
+        hostVals.push("ID: " + host.id);
+    if (host.ip)
+        hostVals.push("IP: " + host.ip);
+
+    const hostInfo = hostVals.join(", ");
+
+    return (<React.Fragment key={index}>
+        <label className='control-label' htmlFor={id}> {`DHCP host ${index + 1}`} </label>
+        <div id={id}> {hostInfo} </div>
+    </React.Fragment>);
+};
+
+export const NetworkOverviewTab = ({ network }) => {
+    const idPrefix = `${networkId(network.name, network.connectionName)}`;
+
+    let ip = [];
+    // Libvirt allows network to have multiple ipv6 and ipv4 addresses.
+    // But we only first one of each
+    ip[0] = network.ip.find(ip => ip.family === "ipv4");
+    ip[1] = network.ip.find(ip => ip.family === "ipv6");
+
+    return (
+        <div className="networks-page-grid">
+            <div className='ct-form-layout'>
+                <label className='control-label label-title'> {_("General")} </label>
+                <span />
+
+                <label className='control-label' htmlFor={`${idPrefix}-persistent`}> {_("Persistent")} </label>
+                <div id={`${idPrefix}-persistent`}> {network.persistent ? _("yes") : _("no")} </div>
+
+                <label className='control-label' htmlFor={`${idPrefix}-autostart`}> {_("Autostart")} </label>
+                <div id={`${idPrefix}-autostart`}> {network.autostart ? _("yes") : _("no")} </div>
+
+                { network.mtu && <React.Fragment>
+                    <label className='control-label' htmlFor={`${idPrefix}-mtu`}> {_("Maximum Transmission Unit")} </label>
+                    <div id={`${idPrefix}-mtu`}> {network.mtu} </div>
+                </React.Fragment> }
+            </div>
+
+            <div className="ct-form-layout">
+                { ip[0] && <React.Fragment>
+                    <label className='control-label label-title'> {_("IPv4 Address")} </label>
+                    <span />
+
+                    { ip[0].address && <React.Fragment>
+                        <label className='control-label' htmlFor={`${idPrefix}-ipv4-address`}> {_("Address")} </label>
+                        <div id={`${idPrefix}-ipv4-address`}> {ip[0].address} </div>
+                    </React.Fragment> }
+
+                    { ip[0].netmask && <React.Fragment>
+                        <label className='control-label' htmlFor={`${idPrefix}-ipv4-netmask`}> {_("Netmask")} </label>
+                        <div id={`${idPrefix}-ipv4-netmask`}> {ip[0].netmask} </div>
+                    </React.Fragment> }
+
+                    { ip[0].dhcp.range.start && <React.Fragment>
+                        <label className='control-label' htmlFor={`${idPrefix}-ipv4-dhcp-range`}> {_("DHCP range")} </label>
```

Comment:
This is a label, so it should use headline case [according to Patternfly](https://www.patternfly.org/styles/terminology-and-wording/#capitalization). Hence it should be DHCP Range, as @KKoukiou suggested.